core/src/main/scala/sttp/tapir/EndpointIO.scala

```diff
@@ -450,15 +451,17 @@ object EndpointIO {
   case class Info[T](
       description: Option[String],
       examples: List[Example[T]],
-      deprecated: Boolean,
+      deprecated: Option[Boolean],
```

Comment:
why is this an option? how would the three values `Some(true)`, `Some(false)`, `None` differ?

Reply:
I thought that it can be good to have 3 translation: